Pcap++/header/PcapFilter.h

```diff
@@ -80,6 +81,7 @@ namespace pcpp
 	protected:
 		bpf_program* m_program;
 		std::string m_lastProgramString;
+		pcpp::LinkLayerType m_lastLinkLayerType;
```

Comment:
Please change the variable name to `m_LastLinkLayerType` (with capital L).

The other variable names should be changed as well, but me or you can do it later.

Reply:
Done.